docs/markdown/Machine-files.md

```diff
@@ -128,6 +128,18 @@ b = a + 'World'
 a = 'Hello'
 ```
 
+*Since 1.3.0* Some tokens are replaced in the machine file before parsing it:
+- `@GLOBAL_SOURCE_ROOT@`: the absolute path to the project's source tree
+- `@DIRNAME@`: absolute path to the file's parent directory.
```

Comment:
Might help to specify machine file's parent directory specifically. Also, either make both definitions start with `the` or not.

Reply:
fixed